hack/ocp-deploy-grafana.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+
+set -eou pipefail
+
+NAMESPACE="${1:-observatorium-logs-test}"
+TENANT_ID="${2:-observatorium}"
+LOKI_PREFIX="${3:-observatorium-loki}"
+
+# Deploy grafana and expose as route
+kubectl --ignore-not-found=true -n "$NAMESPACE" delete deployment grafana
+kubectl create deployment -n "$NAMESPACE" grafana --image=docker.io/grafana/grafana
+oc --ignore-not-found=true -n "$NAMESPACE" delete service grafana
```

Comment:
What's the reason for the inconsistent usage of `kubectl` and `oc`?

Reply:
> What's the reason for the inconsistent usage of `kubectl` and `oc`?

@ronensc no reason ... I moved all to be oc :-)